release-calendar/src/client/stylesheets/app.scss

```diff
@@ -0,0 +1,66 @@
+.main-container {
+    padding: 12px;
+    padding-top: 24px;
+    overflow: hidden;
+}
+
+@mixin container-shadow {
+    box-shadow:
+        rgba(0, 0, 0, 0.2) 0px 2px 4px -1px,
+        rgba(0, 0, 0, 0.14) 0px 4px 5px 0px,
+        rgba(0, 0, 0, 0.12) 0px 1px 10px 0px;
+    padding: 16px;
+    border-radius: 4px;
+}
+
+.col-channel-table {
+    @include container-shadow;
+    padding-top: 0px;
+    width: 30%;
+    float:left;
+}
+
+.col-calendar {
+    @include container-shadow;
+    width: 63%;
+    float:right;
+}
+
+@media only screen and (max-width: 940px) {
+    .col-channel-table {
+        float:left;
+        width: 50%;
+    }
+    .col-calendar {
+        width: 90%;
+        margin-top: 5px;
+        clear: both;
+        float: left;
+    }
+}
+
+@media only screen and (max-width: 600px) {
+    .col-channel-table {
+        width: 70%;
+    }
+}
+
+body {
+    font-family: -apple-system,BlinkMacSystemFont,Segoe UI,Helvetica,Arial,sans-serif,Apple Color Emoji,Segoe UI Emoji;
```

Comment:
I think it's weird to use OS-specific fonts; also, BlinkMacSystemFont and Segoe UI are not web-safe fonts and you're not importing them anywhere with @font-face, so this will look very different depending on the user's system. I'm far from an expert on web fonts though so if you need help figuring this out it's worth asking in the team group chat for help!

Reply:
haha, I didn't make the connection that `apple-system` was "Apple". oops. I stole this font from an amp page. I'll ping someone about it for going about picking a better option.